config/default_proffieboard_config.h

```diff
@@ -68,7 +69,9 @@ Preset presets[] = {
     StyleNormalPtr<BLACK, BLACK, 300, 800>(), "Battery\nLevel"}
 };
 BladeConfig blades[] = {
- { 0, WS2811BladePtr<97, WS2811_800kHz>(),
+ { 0, DimBlade(100.0,WS2811BladePtr<97, WS2811_800kHz>()), // sets initial blade brightness to 100.0% 
```

Comment:
why is this change here?

Reply:
For testing, should have made used own config for this? Will reset for the final version..